web/src/styles/sections/nav-bar.scss

```diff
@@ -20,6 +20,13 @@ $icon-offset: 45px;
   align-items: center;
 }
 
+.reduced-width {
```

Comment:
pebble - can we update the variable name so it's clear it's more specific to the Nac bar

Reply:
sure